options/options.go

```diff
@@ -100,6 +100,9 @@ type TerragruntOptions struct {
 	// The ARN of an IAM Role to assume before running Terraform
 	IamRole string
 
+	// Duration of the STS Session
+	StsDuration int64
```

Comment:
I believe there are `New` and `Clone` methods for `Options` that need to be updated with this param.

Reply:
I've updated the Clone and New method and added a default const for the default StsDuration of 3600.